src/panel/direction/MobileRouteDetails.jsx

```diff
@@ -1,10 +1,29 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import RouteVia from './RouteVia';
 import RoadMap from './RoadMap';
 import { formatDuration } from 'src/libs/route_utils';
+import classnames from 'classnames';
+import throttle from 'lodash.throttle';
 
-const MobileRouteDetails = ({ id, route, origin, destination, vehicle, toggleDetails }) => {
-  return <div className="itinerary_legDetails">
+const MobileRouteDetails =
+({ id, route, origin, destination, vehicle, toggleDetails }) => {
+  const panelElement = useRef(null);
+  const [scrolledDown, setScrolledDown] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = throttle(() => {
+      setScrolledDown(panelElement.current.scrollTop > 0);
+    }, 100, { leading: true, trailing: true });
+    panelElement.current.addEventListener('scroll', handleScroll, { passive: true });
+    return () => window.removeEventListener('scroll', handleScroll);
```

Comment:
Should not that be
```suggestion
    return () => panelElement.current.removeEventListener('scroll', handleScroll);
```
?

Reply:
Oops, indeed. Thank you!